doc/notes_on_celery.py

```diff
@@ -60,31 +60,31 @@ def process_revisions(page_id, page_revs):
 @task_postrun.connect()
 def task_postrun(*args, **kwargs):
     # always, no matter exceptions or not, remove database sessions
-    
-    
+    return
```

Comment:
This is just documentation and it is intended to be interpreted as psuedo-code.  See "listener = " below.